src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Windows/Markup/DependsOnAttribute.cs

```diff
@@ -44,14 +29,9 @@ public DependsOnAttribute(string name)
         }
 
         /// <summary>
-        /// Override of Attribute.TypeId
+        /// Override to ensure AttributeCollection perserves all instances
```

Comment:
I think this should be rewritten to include the previous comment as it is. That seems clear.